packages/anyrender_vello_cpu/src/scene.rs

```diff
@@ -1,39 +1,68 @@
 use anyrender::{NormalizedCoord, Paint, PaintScene};
 use kurbo::{Affine, Rect, Shape, Stroke};
-use peniko::{BlendMode, BrushRef, Color, Fill, Font, StyleRef};
-use vello_cpu::{self, PaintType, Pixmap};
+use peniko::{
+    BlendMode, Brush, BrushRef, Color, Fill, FontData, ImageBrush, ImageSampler, StyleRef,
+};
+use vello_cpu::{ImageSource, PaintType, Pixmap};
+// use vello_cpu::{self, ImageSource, PaintType, Pixmap};
 
 const DEFAULT_TOLERANCE: f64 = 0.1;
 
+fn image_source_to_image_brush(image_source: ImageSource) -> ImageBrush<ImageSource> {
+    ImageBrush::<ImageSource> {
+        image: image_source,
+        sampler: ImageSampler::default(),
+    }
+}
+
 fn brush_ref_to_paint_type<'a>(brush_ref: BrushRef<'a>) -> PaintType {
     match brush_ref {
-        BrushRef::Solid(alpha_color) => PaintType::Solid(alpha_color),
-        BrushRef::Gradient(gradient) => PaintType::Gradient(gradient.clone()),
-        BrushRef::Image(image) => PaintType::Image(vello_cpu::Image::from_peniko_image(image)),
+        Brush::Solid(alpha_color) => PaintType::Solid(alpha_color),
+        Brush::Gradient(gradient) => PaintType::Gradient(gradient.clone()),
+        Brush::Image(image) => {
+            let image_source = vello_cpu::ImageSource::from_peniko_image_data(&image.image);
+            let brush = image_source_to_image_brush(image_source);
+            PaintType::Image(brush)
+        }
```

Comment:
I think this should be something like:

```rust
Brush::Image(image) => {
     let image_source = vello_cpu::ImageSource::from_peniko_image_data(&image.image);
     PaintType::Image(ImageBrush {
           image: image_source,
           sampler: image.sampler,
     })
}
```

We need to use the sampler parameters from the input, and I don't think the additional helper function is buying us anything here.